build.cake

```diff
@@ -32,9 +32,6 @@ var nugetVersion = gitVersionInfo.NuGetVersion;
 ///////////////////////////////////////////////////////////////////////////////
 Setup(context =>
 {
-    if(BuildSystem.IsRunningOnTeamCity)
```

Comment:
IMO this is a very sensible change. We've had several instances where GitVersion's version number was being clobbered by Cake, and a few of those where it was masking the fact that the two disagreed.